changelog.md

```diff
@@ -1,3 +1,13 @@
+1.5.0
+==========
+### New Features:
+**Microsoft Identity Web templates now include a project template for gRPC**. See issue [628](https://github.com/AzureAD/microsoft-identity-web/issues/628) for details.
+**The Microsoft Identity Web B2C templates now use the recommended `.b2clogin.com`** instead of `login.microsoftonline.com` by default. See issue [#792](https://github.com/AzureAD/microsoft-identity-web/issues/792) for details.
+**The Microsoft Identity Web B2C templates now use the Recommended, next-generation, userflows** instead of the Standard user flows by default. See issue [#836](https://github.com/AzureAD/microsoft-identity-web/issues/836) for details.
```

Comment:
```suggestion
**The Microsoft Identity Web B2C templates now use the recommended, next-generation, user flows** instead of the standard user flows by default. See issue [#836](https://github.com/AzureAD/microsoft-identity-web/issues/836) for details.
```

Reply:
leaving them capitalized, as that's how the appear in the b2c portal ui.